zingolib/src/mocks.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(dead_code)]
```

Comment:
I added this because it is going to be a common occurance that mock builders are created in a PR and then used in a following PR. if we remove this it will likely get added again soon so I dont see the point

Reply:
okay